### PR TITLE
Register _vercel.encrypt0r.my.id and _atproto.encrypt0r.my.id

### PR DESCRIPTION
**Pull request by its0din-ai**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [X] The file is in the `domains` folder and is in the JSON format.
- [X] The file's name is all lowercased and alphanumeric. 
- [X] You have completed your website. 
- [X] The website is reachable.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [X] There is sufficient information at the `owner` field.  

## Website Link/Preview

https://encrypt0r.my.id

### CODE REVIEW

**Review by omsenjalia — CHANGES_REQUESTED** [review]
`encrypt0r.is-a.dev` doesn't exist

**Comment by its0din-ai** [comment]
> `encrypt0r.is-a.dev` doesn't exist

domain request added. please let me know if you need something.

File: domains/encrypt0r.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "description": "main domain",
+  "owner": {
+    "username": "encrypt0r",
+    "email": "encrypt0r-x@protonmail.com"
+  },
+  "record": {
+    "A": ['103.16.117.216']
```
Comment (Stef-00012) [inline domains/encrypt0r.json:8]: invalid JSON

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/encrypt0r.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "description": "main domain",
+  "owner": {
+    "username": "encrypt0r",
+    "email": "encrypt0r-x@protonmail.com"
+  },
+  "record": {
+    "A": "103.16.117.216"
```
Comment (Stef-00012) [inline domains/encrypt0r.json:8]: A record should be an array, also i wouldn't consider just this as complete
image

**Review by Stef-DP — CHANGES_REQUESTED** [review]

**Pull request closed without merging**